.github/workflows/deploy-frontend-pr-previews.yml

```
@@ -812,3 +818,159 @@ jobs:
               repo: context.repo.repo,
               body: 'New reports changes available for preview [here](${{ needs.reports.outputs.url }})'
             })
+
+              ### deploy reports preview ###
+  reports:
```

actionlint (1.7.4)

key "reports" is duplicated in "jobs" section. previously defined at line:745,col:3. note that this key is case insensitive

(syntax-check)